### DISCUSSION TITLE
Welcome to PCB-Projects Discussions!

### DISCUSSION THREAD

**Discussion by keirf** [keirf/PCB-Projects#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mandopop-1960Panelman** [comment]
Hi
    Please help. No matter what I do my new greaseweazle V4 always says write protect when I try to write a disk
The disk is not write protected. host tools and fluxengine give the same result and setting the registry USB protect makes no difference,

**Reply by keirf** [reply]
Is the drive you are using known good? The write protect status is detected via a switch, and this switch can go bad over time. The write-protect status check is implemented within firmware: messing with registry entries will do more harm than good.

**Comment by sector101uk** [comment]
Hi Keir,

I have been using your Eprom adaptor a few times and never had a problem with it until I tried recently to read out a few M27C800 Eproms.

There is no revision written on the adaptor, but to help identify the version there are three Jumpers and one push-button 27C322 switch.

Every time I read one of the Eproms into my MiniPro TL886A the results were always different. I could not rely on the resulting file contents being correct.

Long story short, it appears the R1 and R2 10K resistors that pull the jumpered address lines are not strong enough to hold the 800's address pins reliably to get a proper read. I have changed these to 220R (what I had at hand) and the adaptor has been solid with consistent results.

**Reply by keirf** [reply]
Thanks for the report! These resistors would probably be better specified then as 4k7, or perhaps even a bit less than that.
